### PR TITLE
Sort version properties

### PR DESCRIPTION
**Pull request by yurloc**

### CODE REVIEW

**Comment by yurloc** [comment]
Jenkins execute full downstream build.

**Comment by mareknovotny** [comment]
@yurloc i agree with sorting of properties, but it is not right moment as we have multiple PRs open for depednencies and this PR change will impact them to be rebase required, hold off with it please

**Comment by yurloc** [comment]
@mareknovotny OK. Will all those version upgrade PRs merged in the near future?

**Comment by mareknovotny** [comment]
I hope at least for our coming releases. There are Wildfly upgrade and following related WF components PRs

**Comment by yurloc** [comment]
@mareknovotny is this the right time to resume the work on this? I can still see many open PR that upgrade versions.

**Comment by mareknovotny** [comment]
@yurloc I just went through open PR and marked any which has merge conflicts for rebasing ;).

**Comment by mareknovotny** [comment]
Jenkins retest this please

File: pom.xml [hunk 1]
```diff
@@ -46,15 +46,33 @@
...
-    <illegaltransitivereportonly>false</illegaltransitivereportonly>
 
     <!--
       CONVENTIONS:
       - A version property must be specified in the format "version.{groupId}", optionally with a suffix to make it unique.
       - Version properties must be sorted alphabetically (other form of sorting were found to be unclear and ambiguous).
     -->
+
+    <!-- Internal dependencies -->
+    <version.org.kie>7.37.0-SNAPSHOT</version.org.kie>
```
Comment (mareknovotny) [inline pom.xml:57]: we need to change that to the latest SNAPSHOT, 7.40.0-SNAPSHOT

File: pom.xml [hunk 2]
```diff
@@ -160,28 +220,36 @@
     <version.org.codehaus.plexus.plexus-classworlds>2.5.2</version.org.codehaus.plexus.plexus-classworlds>
     <version.org.codehaus.plexus.plexus-containers>1.6</version.org.codehaus.plexus.plexus-containers>
     <version.org.codehaus.plexus.plexus-interpolation>1.21</version.org.codehaus.plexus.plexus-interpolation>
+    <version.org.codehaus.plexus.plexus-io>3.0.0</version.org.codehaus.plexus.plexus-io>
     <version.org.codehaus.plexus.plexus-utils>3.0.22</version.org.codehaus.plexus.plexus-utils>
     <version.org.codehaus.woodstox>4.4.1</version.org.codehaus.woodstox>
+    <version.org.dom4j>2.1.1</version.org.dom4j>
```
Comment (mareknovotny) [inline pom.xml:226]: this was upgraded to 2.1.3

**Comment by mareknovotny** [comment]
I guess we need to carefully rebase or redo this PR as I could see there are old values of properties

**Pull request closed without merging**